tooling-values.yaml

```diff
@@ -1,8 +1,11 @@
 
+sonatype-nexus:
+  enabled: true
+
```

Comment:
Does this need to be here? think it's only needed in the apps/nexus/values?

Reply:
@eformat  - i did a test of this, and it seems to work without this here in my argo on s43. Im getting sync issues cuz of ocp routes though

Reply:
hmm .. nexus just does not enable for me without it ?
```
virt:~/git/ubiquitous-journey ⎇ master#1a40358$ helm2 template  -f tooling-values.yaml charts/ubiquitous-journey | grep nexus
  name: nexus
      releaseName: nexus
        - apps/nexus/values.yaml
virt:~/git/ubiquitous-journey ⎇ master#1a40358$ helm3 template  -f tooling-values.yaml charts/ubiquitous-journey | grep nexus
  name: nexus
      releaseName: nexus
        - apps/nexus/values.yaml
``` 

requirements.yaml -> condition: sonatype-nexus.enabled
values.yaml -> sonatype-nexus.enabled: false
charts/ubiquitous-journey/apps/nexus/values.yaml -> sonatype-nexus.enabled: true

so .. how is the requirements condition set ?